Source/WebDriver/WebDriverService.cpp

```diff
@@ -2721,9 +2722,9 @@ void WebDriverService::bidiSessionStatus(unsigned id, RefPtr<JSON::Object>&&, Fu
 void WebDriverService::bidiSessionSubscribe(unsigned id, RefPtr<JSON::Object>&&parameters, Function<void(WebSocketMessageHandler::Message&&)>&& completionHandler)
 {
     // https://w3c.github.io/webdriver-bidi/#command-session-subscribe
-    auto eventNames = parameters->getArray("events"_s);
+    auto eventNamesJS = parameters->getArray("events"_s);
```

Comment:
Nit: technically t's not JS, it's an untyped JSON array (`JSON::Array<JSON::Value>`)